ktor-client/ktor-client-tests/common/test/io/ktor/client/tests/utils/Logging.kt

```diff
@@ -24,6 +32,7 @@ internal class TestLogger(private vararg val expectedLog: String) : Logger {
         var actualIndex = 0
 
         val message = StringBuilder()
+        val stashed = ArrayList<String>()
```

Comment:
minor: can be set

Reply:
This is a list intentionally: you can specify the same text multiple times